model-formulation/job_mix_creator.py

```diff
@@ -0,0 +1,231 @@
+DATA_STORE = DATA_STORE = '/home/ps35324/gpu-sharing-scheduler/src/data-store/scheduler.db'
```

Comment:
Path again. Let this be an argument

Reply:
Hmmm, this database contains the data I collected in the previous TieBreaker iteration (over 500K datapoints). Part of it includes the profiles of running each job on each slice in MIG. The file is too large to include in GitHub. For the final artifact, we should have this data formatted and accessible in a better format. For the time being, I'll put a relative path or comment it out completely, as we don't need to run this code any longer to make progress to the deadline.